osu.Game.Tests/Visual/UserInterface/TestSceneUpdateableBeatmapSetCover.cs

```diff
@@ -57,11 +57,15 @@ public void TestUnloadAndReload()
                     }
                 };
 
+#if NET5_0
                 var coverTypes = Enum.GetValues<BeatmapSetCoverType>();
+#else
+                var coverTypes = Enum.GetValues(typeof(BeatmapSetCoverType)).Cast<BeatmapSetCoverType>();
+#endif
```

Comment:
i'd probably advise *not* using conditional compilation symbols unless absolutely necessary.

Reply:
So in these cases, always going old-style is preferrable?

Reply:
if feasible, yes. conditional compilation symbols are annoying to work with, as they work at source-level (as in, if a conditional is not active, the code within is fully dead, and won't get syntax highlighting/compile checking; think of it as if it just wasn't there) - so now you have to build for all targets to make sure the project always builds.